android/java/org/chromium/chrome/browser/crypto_wallet/activities/BuySendSwapActivity.java

```diff
@@ -268,29 +485,27 @@ private int getIndexOf(Spinner spinner, String chainId) {
     private void adjustControls() {
         EditText toValueText = findViewById(R.id.to_value_text);
         TextView marketPriceValueText = findViewById(R.id.market_price_value_text);
-        TextView slippingToleranceValueText = findViewById(R.id.slipping_tolerance_value_text);
-        RadioGroup radioBuySendSwap = findViewById(R.id.buy_send_swap_type_radio_group);
+        // RadioGroup radioBuySendSwap = findViewById(R.id.buy_send_swap_type_radio_group);
```

Comment:
is this commented variable left by purpose?

Reply:
Yes, it's for market/limit orders, we do it without that for now, but could be needed in the future

Reply:
thanks :+1: